ffig/filters/capi_filter.py

```diff
@@ -51,7 +51,7 @@ def to_go(t):
         if t.pointee.kind == TypeKind.CHAR_S:
             return 'string'
         if t.pointee.kind == TypeKind.RECORD:
-            return 'unsafe.Pointer'
+            return t.pointee.name.replace('const', '')
```

Comment:
A space after const? It doesn't matter since the result will just contain an extra space. I think we should be using automated code formatting tools to make the template output look readable, not relying on fragile special-casing in the generators.